From be3c927894ecf1455a1a9599ff18421f1e34c99b Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Sun, 8 Nov 2020 11:34:50 +0100
Subject: [PATCH] transmission-web-control: remove Transmission SSL variants

In recent commits, there were removed Transmission SSL variants and
there is just used one variant of transmission-daemon. Let's adjust it here as well.

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 net/transmission-web-control/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/net/transmission-web-control/Makefile b/net/transmission-web-control/Makefile
index 32d554faf..f9342217e 100644
--- a/net/transmission-web-control/Makefile
+++ b/net/transmission-web-control/Makefile
@@ -1,7 +1,7 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=transmission-web-control
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://github.com/ronggang/transmission-web-control
@@ -19,7 +19,7 @@ define Package/transmission-web-control
   SECTION:=net
   CATEGORY:=Network
   SUBMENU:=BitTorrent
-  DEPENDS:=@(PACKAGE_transmission-daemon-openssl||PACKAGE_transmission-daemon-mbedtls)
+  DEPENDS:=+transmission-daemon
   CONFLICTS:=transmission-web
   TITLE:=Transmission Web Control
   URL:=https://github.com/ronggang/transmission-web-control
-- 
2.25.1

